### DISCUSSION TITLE
Artifacts, create a multi-level value system of artifacts to be captured by the AI [Done]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#4592] [Ideas]
It would be interesting to create a multi-level value system of artifacts to be captured by the AI.
Some are very important to capture and some are less important.

**Comment by ihhub** [comment]
Hi @LeHerosInconnu , I think you're referring to #4226 and surely you're welcome to contribute. I can write down how it looks like right in the code. I'll update the issue.

**Reply by LeHerosInconnu** [reply]
Hello @ihhub,

> Hi @LeHerosInconnu , I think you're referring to #4226 and surely you're welcome to contribute. I can write down how it looks like right in the code. I'll update the issue.

Oh, I read this issue and didn't remember it. :)
Please, reduce the number of issues to something like 100. :rofl:

**Comment by LeHerosInconnu** [comment]
This has been implemented.
I close the discussion.
